src/api/spec/models/notification_spec.rb

```diff
@@ -33,4 +33,42 @@
 
     it { expect(test_group.notifications).to include(notification) }
   end
+
+  describe '#user_active?' do
+    let(:rss_notification) { create(:rss_notification, subscriber: test_user) }
+    subject { rss_notification.user_active? }
+
+    context 'when subscriber is away' do
+      let(:test_user) { create(:dead_user, login: 'foo') }
+
+      it { expect(subject).to be_falsey }
```

Comment:
:thought_balloon: Any reason to not use `be false`?

Reply:
rubocop

Reply:
ow, ok